dev/NavigationView/NavigationView.idl

```diff
@@ -159,6 +159,7 @@ unsealed runtimeclass NavigationView : Windows.UI.Xaml.Controls.ContentControl
     [MUX_DEFAULT_VALUE("1008.0")]
     [MUX_PROPERTY_VALIDATION_CALLBACK("CoerceToGreaterThanZero")]
     Double ExpandedModeThresholdWidth { get; set; };
+    Windows.Foundation.Collections.IVector<Object> FooterMenuItems{ get; };
```

Comment:
@yulikl@microsoft.com I think this has to go through the API review process?

Reply:
It definitely does. @ad1Dima in the mean time can you please add the preview tag to this new API?